validators/comment.validator.js

```diff
@@ -1,5 +1,16 @@
 const Joi = require('joi');
 
+const commentValidator = Joi.object({
+  text: Joi.string()
+    .trim()
+    .min(2)
+    .max(700),
```

Comment:
Why is there a max restriction of 700 characters?
I'd recommend having no limit or have a reasonable limit like 100 000 characters or so.

Reply:
fixed